package/gluon-setup-mode/files/lib/gluon/setup-mode/led.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+# shellcheck disable=SC1091,SC2154
+
+get_setup_mode_led() {
+	. /etc/diag.sh
+	get_status_led 2> /dev/null
+
+	if [ -z "$status_led" ]; then
+		status_led="$running"
+	fi
+
+	if [ -z "$status_led" ]; then
+		status_led="$boot"
+	fi
+
+	custom_led="$(lua -e 'print(require("gluon.setup-mode").get_status_led() or "")')"
+	if [ -z "$status_led" ]; then
+		status_led="$custom_led"
+	fi
+}
+
+MODE="$1"
+if [ "$MODE" = "running" ]; then
+	# runing - slow pulse
+	get_setup_mode_led 2> /dev/null
```

Comment:
Why is only one of these redirecting stderr?

Reply:
This is an oversight.